updateRuntimeVersions.sh

```diff
@@ -23,6 +23,7 @@ echo ">>> Updating Dependencies.scala"
 sed -i.bak "s/sys.props.getOrElse(\"akka-runtime.version\", \"\(.*\)\")/sys.props.getOrElse(\"akka-runtime.version\", \"$RUNTIME_VERSION\")/" ./project/Dependencies.scala
 rm ./project/Dependencies.scala.bak
 
-echo ">>> Updating akka-javasdk-maven/akka-javasdk-parent/pom.xml"
-sed -i.bak "s/<akka-runtime\.version>[^<]*/<akka-runtime.version>$RUNTIME_VERSION/" ./akka-javasdk-maven/akka-javasdk-parent/pom.xml
-rm ./akka-javasdk-maven/akka-javasdk-parent/pom.xml.bak
+# The runtime version in the parent pom.xml is automatically replaced by `sbt publishM2`, but let's keep it in sync
+echo ">>> Updating akka-javasdk-parent/pom.xml"
+sed -i.bak "s/<akka-runtime\.version>[^<]*/<akka-runtime.version>$RUNTIME_VERSION/" ./akka-javasdk-parent/pom.xml
+rm ./akka-javasdk-parent/pom.xml.bak
```

Comment:
Is this needed anymore if sbt postprocesses the pom.xml using the right runtime version?